2주차/1620.py

```diff
@@ -0,0 +1,23 @@
+import sys
+input = sys.stdin.readline
+
+N, M = map(int, input().split())
+character = {}
+num = 1
+quiz = []
+
+for i in range(N + M):
+    S = input().strip()
+    if i < N:
+        character[num] = S
+        num += 1
+    else:
+        quiz.append(S)
```

Comment:
```suggestion
for i in range(N):
    S = input().strip()
    character[num] = S
    num += 1

for i in range(M):
    S = input().strip()
    quiz.append(S)
```
이렇게 두 for문으로 분리해주는게 훨씬 코드 가독성도 좋아지고, 쓸데없는 i<N 비교도 안해서 속도도 빨라질 것 같아요.


```
character = {i+1: input().strip() for i in range(N)}
quiz = [input().strip() for _ in range(M)]
```
이런식으로 딕셔너리 컴프리헨션, 리스트 컴프리헨션 써주면 더 좋고요